docs/help/tklinuxcnc.adoc

```diff
@@ -276,84 +276,86 @@ Manufacturing Sciences (NCMS) on the Next Generation Controller (NGC)
 project. The dialect is similar to the Fanuc series of CNCs.
 
 .G-codes
-----
-G00 X Y Z          rapid feed
-G01 X Y Z          linear feed, e.g., G01 X10 Y20 Z30 F60 for 60 ipm
-G02 X Y Z I J      clockwise circular feed (helical with Z)
-G03 X Y Z I J      counterclockwise circular feed (helical with Z)
-G04 P<#>           dwell # of seconds
-G10 L2 P<#> X Y Z  set coordinates for system #1-9, 1 = G54, 9 = G59.3
-G17                XY plane circular interpolation
-G18                ZX plane circular interpolation
-G19                YZ plane circular interpolation
-G20                English units (inches)
-G21                metric units (millimeters)
-G40                cancel cutter compensation
-G41 D<#>           cutter compensation for tool <#>, tool on left
-G42 D<#>           cutter compensation for tool <#>, tool on right
-G43 H<#>           use tool length offset for tool #, 0 = no offset
-G49                cancel tool length offset
-G53                turn any coordinate offsets off for this block
-G54                use coordinate offsets for system #1, from G10 L2 P1
-G55                "" for system #2, from G10 L2 P2
-G56                "" for system #3, from G10 L2 P3
-G57                "" for system #4, from G10 L2 P4
-G58                "" for system #5, from G10 L2 P5
-G59                "" for system #6, from G10 L2 P6
-G59.1              "" for system #7, G10 L2 P7
-G59.2              "" for system #8, G10 L2 P8
-G59.3              "" for system #9, G10 L2 P9
-G61                exact stop mode
-G64                cutting mode
-G76                threading canned cycle
-G80                cancel fixed (canned) cycle
-G81                drilling cycle, no dwell, rapid out
-G82                drilling cycle, dwell, rapid out
-G83                deep hole peck drilling cycle
-G84                tapping cycle
-G85                boring cycle, no dwell, feed out
-G86                boring cycle, spindle stop, rapid out
-G87                back boring cycle
-G88                boring cycle, spindle stop, manual out
-G89                boring cycle, with dwell, feed out
-G90                non-incremental programming (the usual)
-G91                incremental programming
-G93                inverse time feed mode
-G94                feed per minute mode (the usual)
-G98,G99            set canned cycle return level
-----
+[cols="1,1"]
+|===
+|G00 X Y Z         | rapid feed |
```

Comment:
Good idea to have this as table, but:
For all lines I think the last "|" is too much. At least according to the docs and my preview plug-in is also confused with this.
```suggestion
|G00 X Y Z         | rapid feed
```

And also for the following table.

Reply:
Fixed - and also installed a previewer.